prime-router/src/test/kotlin/MapperTests.kt

```diff
@@ -309,6 +309,92 @@ class MapperTests {
         assertThat(actual).isEqualTo(expected)
     }
 
+    @Test
+    fun `test lookup with NpiLookupMapper`() {
+        /* Schema and table configuration for tests */
+        val table = LookupTable.read("./src/test/resources/metadata/tables/npi-lookup.csv")
+        val tableName = "npi-lookup"
+        val schema = Schema(
+            tableName, topic = "Testing NPI lookup mapper",
+            elements = listOf(
+                Element(
```

Comment:
sonar cloud is complaining here because these lines of code are very similar to 158 and 178. you may have to do a "for each loop trick" to create these `Element`s without "duplication"

Reply:
or, create a method in this test file that creates `Element`s for testing that we can use to de-dup code.

Reply:
Took a stab at it, we'll see what SonarCloud thinks 😬